src/pages/index.js

```diff
@@ -153,9 +288,21 @@ function closeModal(modal) {
 
 function saveProfileChanges(evt) {
   evt.preventDefault();
-  profileName.textContent = modalName.value;
-  profileDescription.textContent = modalDescription.value;
-  closeModal(editProfileModal);
+
+  const submitBtn = evt.submitter;
+  submitBtn.textContent = "Saving...";
+
+  api
+    .editUserInfo({ name: modalName.value, about: modalDescription.value })
+    .then((data) => {
+      profileName.textContent = data.name;
+      profileDescription.textContent = data.about;
+      closeModal(editProfileModal);
+    })
+    .catch(console.error)
+    .finally(() => {
+      submitBtn.textContent = "Save";
```

Comment:
Please, reuse ` renderLoading(false, submitBtn);` when you change the profile data as well not to duplicate the code